pkg/trace/stats/aggregation.go

```diff
@@ -83,12 +85,30 @@ func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregation
 	if enablePeerSvcAgg {
 		agg.PeerService = s.Meta[tagPeerService]
 	}
+	agg.ExtraTagsKey = extraTagsKey(s, extraTags)
 	return agg
 }
 
+func extraTagsKey(s *pb.Span, extraTags []string) string {
+	if len(extraTags) == 0 {
+		return ""
+	}
+	// TODO: replace with strings.Builder?
```

Comment:
If we stick this this approach, we indeed probably want to use a `strings.Builder`, sort the tags, and choose a less-common character as a delimiter. (And potentially escape delimiters that are present in the tags)

Reply:
:+1: The sorting is handled already during the initialization of the concentrator.